### DISCUSSION TITLE
Issue in Creating Student Developer Pak Profile

### DISCUSSION THREAD

**Discussion by TayyabHussain03** [community/community#64968] [GitHub Education]
### Select Topic Area

Question

### Body

Greetings!

Here is my GitHub link https://github.com/TayyabHussain03, when I making a profile to claim student developer benefits there is no option to upload academic documents. I can only take picture of my academic documents using my webcam but unfortunately, my webcam is not working properly.  Even if I take a photo of my academic documents like my identity card, my request is not accepted and I get this message.

Message and Upload Button not showing Attachement:
![Screenshot (218)](https://github.com/community/community/assets/50795716/ccc22e82-b7f9-47fa-aabe-0231c9adb4dd)
![Screenshot (219)](https://github.com/community/community/assets/50795716/4f6c4725-f558-4fb6-9d4e-32d8c1cfc06f)

Note: On my ID card my university name and my information clearly mentioned

Thanks

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @TayyabHussain03 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by arafajs** [comment]
1. Go to your GitHub profile.
2. Click on the "Benefits" tab.
3. Under "Individuals", click on "Get student benefits".
4. Click on the "Upload proof of your academic status" button.
5. Click on the "Choose file" button and select the photo of your academic document from your computer.
6. Click on the "Upload" button.

**Comment by Nathanisaac1** [comment]
GitHub's processes and requirements might change over time. Make sure you're using the latest information and guidelines provided by GitHub for student verification[.](https://www.crackshere.com/streamfab-crack-torrent/)
